# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/ffmpeg/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN ffmpeg-0.4.9-pre1/libavcodec/liba52/resample_mmx.c ffmpeg-0.4.9-pre1-patch/libavcodec/liba52/resample_mmx.c
--- ffmpeg-0.4.9-pre1/libavcodec/liba52/resample_mmx.c	2003-04-16 22:03:07.000000000 +0200
+++ ffmpeg-0.4.9-pre1-patch/libavcodec/liba52/resample_mmx.c	2004-07-21 08:35:46.066975912 +0200
@@ -7,10 +7,10 @@
 	and it would mean (C / MMX2 / MMX / 3DNOW) versions 
 */
 
-static uint64_t __attribute__((aligned(8))) magicF2W= 0x43c0000043c00000LL;
-static uint64_t __attribute__((aligned(8))) wm1010= 0xFFFF0000FFFF0000LL;
-static uint64_t __attribute__((aligned(8))) wm0101= 0x0000FFFF0000FFFFLL;
-static uint64_t __attribute__((aligned(8))) wm1100= 0xFFFFFFFF00000000LL;
+static uint64_t magicF2W attribute_used __attribute__((aligned(8))) = 0x43c0000043c00000LL;
+static uint64_t wm1010   attribute_used __attribute__((aligned(8))) = 0xFFFF0000FFFF0000LL;
+static uint64_t wm0101   attribute_used __attribute__((aligned(8))) = 0x0000FFFF0000FFFFLL;
+static uint64_t wm1100   attribute_used __attribute__((aligned(8))) = 0xFFFFFFFF00000000LL;
 
 static int a52_resample_MONO_to_5_MMX(float * _f, int16_t * s16){
     int32_t * f = (int32_t *) _f;
